src/security-server/admin-service/infra-jpa/src/main/resources/liquibase/serverconf/005-configuration-properties.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.19.xsd">
+
+    <changeSet author="niis" id="005-configuration-properties">
+        <createTable tableName="configuration_properties">
+            <column name="id" autoIncrement="true" type="SERIAL">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="property_key" type="VARCHAR(512)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="property_value" type="VARCHAR(512)">
```

Comment:
512 might be bit too low if we want to future-proof this, lets put a bigger value here, 4k perhaps?